src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -63,6 +63,11 @@ See the LICENSE file in the project root for more information.
       <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Security.Cryptography.Native.OpenSsl.a" Condition="'$(TargetOS)' != 'OSX'"/>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetOS)' == 'OSX'">
+      <AdditionalFramework Include="CoreFoundation" />
+      <AdditionalFramework Include="Security" />
+    </ItemGroup>
+
     <ItemGroup>
       <LinkerArg Include="@(NativeLibrary)" />
       <LinkerArg Include="@(AdditionalNativeLibrary)" />
```

Comment:
BTW: Do you happen to know why we have both `NativeLibrary` and `AdditionalNativeLibrary`? What is the Additional trying to distinguish?

Reply:
This got introduced in #2411. It was a huge change and I don't think anyone noticed it at that point. @sharah?

IMHO we should stick to `NativeLibrary` (and `NativeFramework`). I can do the change to `NativeLibrary` in a separate pull request if there's no good reason to keep the distinction.